src/gmastertrack.cxx

```diff
@@ -269,7 +274,7 @@ GMasterTrack::GMasterTrack(int x, int y, int w, int h, const char* l ) :
 void GMasterTrack::setBpm( int b )
 {
 	bpm = b;
-	tempoDial.value( ( bpm - 60 ) / 160.f );
+	tempoDial.value( ( bpm - MIN_TEMPO ) / (float)(MAX_TEMPO-MIN_TEMPO) );
```

Comment:
Nitpick: missing spaces around the - sign here in MAX - MIN

Reply:
fixed.